### ISSUE TITLE
Orocommerce Maintenance (Impossible to do this action)

### ISSUE THREAD

**Issue by mrdgama** [lexik/LexikMaintenanceBundle#76]
Hello masters,

 I would like to ask the question, is there anybody here who facing the problem when maintenance Orocommerce (upgrade the last version) -- > when do the command : **app/console lexik:maintenance:unlock --env prod**
got the warning "Impossible to do this action". Please help me if anyone solved.

Thanks

**Comment by aivus** [comment]
Hi @mrdgama 

In general it means that your system was not locked with `lexik:maintenance:lock`.

Try to call `app/console lexik:maintenance:lock --env prod` before.

**Comment by m-vo** [comment]
I'm facing the same issue - actually the system got locked via `lexik:maintenance:lock`.

**Comment by aivus** [comment]
@m-vo what is the result of `lexik:maintenance:lock`? Which driver are you using? File?

**Comment by m-vo** [comment]
@aivus Yes, I'm using the `FileDriver`. 

The following is shown when locking:   

    Ttl doesn't work with Lexik\Bundle\MaintenanceBundle\Drivers\FileDriver driver

**Comment by aivus** [comment]
@m-vo 

> The following is shown when locking:
> `Ttl doesn't work with Lexik\Bundle\MaintenanceBundle\Drivers\FileDriver driver`

Yes. it's ok. It just showing that `FileDriver` doesn't support [TTL feature](https://github.com/lexik/LexikMaintenanceBundle/blob/d89f7851682608f0baa22ca557e66d93f19f8232/Resources/doc/index.md#commands)

Could you please share you lexik config? Which symfony version are you using?
I tried to reproduce this, but couldn't.

**Comment by m-vo** [comment]
Sure, we're using `LexikMaintenaceBundle` inside [Contao Open Source CM](https://github.com/contao)S. I did not change the default config:

https://github.com/contao/manager-bundle/blob/4.5/src/Resources/skeleton/app/config.yml#L68-L75
